### DISCUSSION TITLE
请问有什么推荐的搭建服务器的网站？

### DISCUSSION THREAD

**Discussion by ShiomiyaMyo** [Meloong-Git/PCL#1611] [提问] [answered]
最近想跟朋友开个整合包服务器玩，看了一些教程大多数都要用阿里云之类的但是听说这些不大好...之前有用过pebblehost但是感觉一般，请问有什么其他这种类型的网站推荐吗，或者有什么比用这些网站更方便的搭建服务器方法？ 

还有，请问能在forge上下载的整合包服务器版本要怎么使用？

**Comment by Chiloven945** [comment, marked as answer]
可以用vps，这个服务商挺多

**Reply by ShiomiyaMyo** [reply]
好滴谢谢

**Reply by Chiloven945** [reply]
我说vps是虚拟服务器的意思哈不是一个什么品牌，很多家都有的你可以自己看一看
